src/main/java/appeng/core/features/AEFeature.java

```diff
@@ -42,9 +42,13 @@ public boolean isVisible()
 	GrindStone( Constants.CATEGORY_WORLD ),
 	Flour( Constants.CATEGORY_WORLD ),
 	Inscriber( Constants.CATEGORY_WORLD ),
+	Charger( Constants.CATEGORY_WORLD ),
+	CrystalGrowthAccelerator( Constants.CATEGORY_WORLD ),
 	ChestLoot( Constants.CATEGORY_WORLD ),
 	VillagerTrading( Constants.CATEGORY_WORLD ),
 	TinyTNT( Constants.CATEGORY_WORLD ),
+	CertusOre( Constants.CATEGORY_WORLD ),
+	ChargedCertusOre( Constants.CATEGORY_WORLD ),
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS00115)